ShellCheck/Formatter/TTY.hs

```diff
@@ -44,13 +44,13 @@ colorForLevel level =
         "message" -> 1 -- bold
         "source"  -> 0 -- none
         otherwise -> 0 -- none
-    
+
 outputError file error = do
-    color <- getColorFunc
-    hPutStrLn stderr $ color "error" $ file ++ ": " ++ error
+    color <- getColorFunc $ ColorAuto -- FIXME: should respect --color
+    hPutStrLn stderr $ color "error ZZZ" $ file ++ ": " ++ error
```

Comment:
Whoops, I accidentally left this in. I'll fix it as soon as I get access to a proper computer.